skqulacs/circuit/circuit.py

```diff
@@ -500,6 +507,35 @@ def _add_parametric_R_gate_inner(
 
         return parameter_id
 
+    def _add_multi_qubit_parametric_R_gate_inner(
```

Comment:
This function is not called. Why do you defined this function?
if it isn't nessesary, delete it.

Reply:
This can be used to define 2qubit parametric gates. In the repository there was already something similar, but now is completed. Is not used in codes, but allows to recreate easily some ansatz with 2-qubit parametric gates.

Reply:
I actually use it in some personal codes DX

Reply:
Can you integration `add_parametric_multi_Pauli_rotation_gate()` function?

Reply:
OK, I will insert another circuit in the test file and create an ansatz with the function.